From: John Zupin <johnz@lunarg.com>
Date: Thu, 23 Mar 2023 15:29:57 -0600
Subject: patch: update to use jsoncpp so

---
 layer/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/layer/CMakeLists.txt b/layer/CMakeLists.txt
index 5f66b9d..1abc219 100644
--- a/layer/CMakeLists.txt
+++ b/layer/CMakeLists.txt
@@ -89,7 +89,7 @@ target_sources(VkLayer_khronos_profiles PRIVATE
 target_link_libraries(VkLayer_khronos_profiles PRIVATE
     Vulkan::LayerSettings
     Vulkan::Headers
-    jsoncpp_static
+    jsoncpp_lib
     valijson
 )
 
