From: John Zupin <johnz@lunarg.com>
Date: Tue, 18 Oct 2022 15:13:37 -0600
Subject: patch: don't fetch tinyxml2

---
 CMakeLists.txt | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3b94103..5b3cca4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,12 +72,12 @@ if(REGENERATE_JSONCPP)
     )
 endif()
 
-FetchContent_Declare(
-    tinyxml2
-    GIT_REPOSITORY https://github.com/leethomason/tinyxml2.git
-    GIT_TAG 9.0.0
-)
-FetchContent_MakeAvailable(tinyxml2)
+# FetchContent_Declare(
+#     tinyxml2
+#     GIT_REPOSITORY https://github.com/leethomason/tinyxml2.git
+#     GIT_TAG 9.0.0
+# )
+# FetchContent_MakeAvailable(tinyxml2)
 
 set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
 set(BUILD_OBJECT_LIBS OFF CACHE BOOL "" FORCE)
