From: John Zupin <johnz@lunarg.com>
Date: Tue, 7 Mar 2023 09:28:21 -0700
Subject: patch: update libjsoncpp lib cmake name

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 378d371..7d4b424 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -212,7 +212,7 @@ if (WIN32)
 else()
     macro(add_vk_layer target)
         add_library(VkLayer_${target} SHARED ${ARGN})
-        target_link_Libraries(VkLayer_${target} vku Vulkan::Headers jsoncpp_static valijson vku)
+        target_link_Libraries(VkLayer_${target} vku Vulkan::Headers jsoncpp_lib valijson vku)
         if(ANDROID)
             # Android needs -llog for __android_print_log()
             target_link_Libraries(VkLayer_${target} log)
