From: John Zupin <johnz@lunarg.com>
Date: Mon, 17 Jan 2022 14:03:42 -0700
Subject: packaging patch: fix path to vk.xml file

---
 library/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/library/CMakeLists.txt b/library/CMakeLists.txt
index 20c84ac..a5e3a8e 100644
--- a/library/CMakeLists.txt
+++ b/library/CMakeLists.txt
@@ -7,5 +7,5 @@ add_library(Vulkan::Profiles ALIAS VulkanProfiles)
 source_group("Python Files" FILES ${CMAKE_CURRENT_SOURCE_DIR}/scripts/genvp.py)
 
 add_custom_target(library_profiles
-	COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/library/scripts/genvp.py -registry ${vulkan-headers_SOURCE_DIR}/registry/vk.xml -profiles ${CMAKE_CURRENT_SOURCE_DIR}/../profiles -outIncDir ${CMAKE_CURRENT_SOURCE_DIR}/include/vulkan -outSrcDir ${CMAKE_CURRENT_SOURCE_DIR}/source
+	COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/library/scripts/genvp.py -registry ${vulkan-headers_SOURCE_DIR}/share/vulkan/registry/vk.xml -profiles ${CMAKE_CURRENT_SOURCE_DIR}/../profiles -outIncDir ${CMAKE_CURRENT_SOURCE_DIR}/include/vulkan -outSrcDir ${CMAKE_CURRENT_SOURCE_DIR}/source
 	SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/scripts/genvp.py)
