From: John Zupin <johnz@lunarg.com>
Date: Mon, 17 Jan 2022 14:19:48 -0700
Subject: packaging patch: dont algamate jsoncpp repo

---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 80b3423..e22f897 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,9 +71,9 @@ set(BUILD_OBJECT_LIBS OFF CACHE BOOL "" FORCE)
 # endif()
 
 # After all the content is declared, it can safely be made available.
-FetchContent_MakeAvailable(googletest)
-FetchContent_MakeAvailable(jsoncpp)
-execute_process(COMMAND ${PYTHON_EXECUTABLE} ${jsoncpp_SOURCE_DIR}/amalgamate.py WORKING_DIRECTORY ${jsoncpp_SOURCE_DIR})
+# FetchContent_MakeAvailable(googletest)
+# FetchContent_MakeAvailable(jsoncpp)
+# execute_process(COMMAND ${PYTHON_EXECUTABLE} ${jsoncpp_SOURCE_DIR}/amalgamate.py WORKING_DIRECTORY ${jsoncpp_SOURCE_DIR})
 
 set(UPDATE_DEPS ON)
 set(BUILD_TESTS OFF)
