From: John Zupin <johnz@lunarg.com>
Date: Mon, 7 Feb 2022 16:34:57 -0700
Subject: patch: disable scripts directory

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e22f897..ff0582d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,7 @@ set(VULKAN_TAG v1.3.204)
 include(GNUInstallDirs)
 
 # Using Python 3.7.2 or better (required for typing.OrderedDict)
-find_package(PythonInterp 3.7.2 REQUIRED)
+#find_package(PythonInterp 3.7.2 REQUIRED)
 
 # Third-party libraries
 include(FetchContent)
@@ -154,7 +154,7 @@ set(API_NAME "Vulkan" CACHE STRING "API name to use when building")
 string(TOLOWER ${API_NAME} API_LOWERCASE)
 add_definitions(-DAPI_NAME="${API_NAME}")
 
-add_subdirectory(scripts)
+#add_subdirectory(scripts)
 
 add_subdirectory(library)
 
